src/App.jsx

```diff
@@ -1,44 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Improve imports order 

Useful link:
https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf